core-libs/setup/ssr/logger/loggers/server-logger.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { InjectionToken } from '@angular/core';
+import { Request } from 'express';
+
+export interface LogContext {
+  request?: Request | any;
+  options?: any;
+}
+
+export class ServerLogger {
+  log(message: string, _context?: LogContext): void {
+    /* eslint-disable-next-line no-console */
+    console.log(message);
+  }
+  warn(message: string, _context?: LogContext): void {
+    /* eslint-disable-next-line no-console */
+    console.warn(message);
+  }
+  error(message: string, _context?: LogContext): void {
+    /* eslint-disable-next-line no-console */
+    console.error(message);
+  }
+}
+
+export const serverLoggerToken = new InjectionToken<ServerLogger>(
+  'ServerLogger'
+);
+
+export const loggerEnabled = new InjectionToken<boolean>('featureFlag.logger');
```

Comment:
[MINOR]

Do I understand correctly, that this flag controls only the behavior of the logger in Server (but not CSR)? To be precise, does this flag enable **adding context** to the logs in the server?

If yes, then `loggerEnabled` name might be a bit misleading.
What about `ENABLE_CONTEXTUAL_SERVER_LOGGER`?

PS I suggest to stick to the convention that injection tokens are in CONSTANT_CASE format.